src/runtime_src/core/edge/drm/zocl/zocl_drv.c

```diff
@@ -1065,16 +1081,19 @@ static int zocl_drm_platform_probe(struct platform_device *pdev)
 
 	platform_set_drvdata(pdev, zdev);
 
-	sscanf(XRT_DRIVER_VERSION, "%d.%d.%d",
-		&zocl_driver.major,
-		&zocl_driver.minor,
-		&zocl_driver.patchlevel);
-	sscanf(XRT_DATE, "%d-%d-%d ", &year, &mon, &day);
-        //e.g HASH_DATE ==> Wed, 4 Nov 2020 08:46:44 -0800
-        //e.g XRT_DATE ==> 2020-11-04
-	snprintf(driver_date, sizeof(driver_date),
-		"%d%02d%02d", year, mon, day);
-
+	/* Work around for CR-1119382 issue.
+	 * ZOCL driver is crashing if it accessing the device tree node */
+	if (ZOCL_PLATFORM_ARM64) {
```

Comment:
This change leads to a NULL-ptr access in drmCopyVersion when running on AARCH32. Please revert if possible